src/app/containers/CpsAssetMediaPlayer/index.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  shouldMatchSnapshot,
+  isNull,
+  suppressPropWarnings,
+} from '@bbc/psammead-test-helpers';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { ToggleContext } from '#contexts/ToggleContext';
+import CpsAssetMediaPlayerContainer from '.';
+import {
+  validAresMediaVideoBlock,
+  defaultToggles,
+} from '../MediaPlayer/fixtureData';
+
+const GenerateMediaPlayer = ({
+  /* eslint-disable react/prop-types */
+  platform,
+  blocks,
+  assetUri,
+  /* eslint-disable react/prop-types */
```

Comment:
Does this syntax disable eslint for a specific block of code?  Haven't seen it before